qiime2/sdk/tests/test_method.py

```diff
@@ -444,14 +450,102 @@ def test_async_with_multiple_outputs(self):
         self.assertEqual(result.right.view(list), [-2, 43, 6])
 
     def test_docstring(self):
-        method = self.plugin.methods['split_ints']
-        exp = "QIIME 2 Method"
-        self.assertEqual(exp, method.__doc__)
-        obs = method.__call__.__doc__.split('\n')
-        self.assertTrue(obs[0].startswith('Split sequence of integers'))
-        self.assertEqual(len(obs[1]), 0)
-        self.assertTrue(obs[2].startswith('This method splits'))
-        self.assertTrue(obs[3].startswith('(left and'))
+        merge_mappings = self.plugin.methods['merge_mappings']
+        split_ints = self.plugin.methods['split_ints']
+        identity_with_optional_metadata = (
+            self.plugin.methods['identity_with_optional_metadata'])
+        no_input_method = self.plugin.methods['no_input_method']
+        params_only_method = self.plugin.methods['params_only_method']
+
+        self.assertEqual(merge_mappings.__doc__, 'QIIME 2 Method')
+
+        merge_calldoc = merge_mappings.__call__.__doc__
+        self.assertEqual(exp_merge_calldoc, merge_calldoc)
+        self.assertIn("No description available", merge_calldoc)
+
+        split_ints_return = split_ints.__call__.__doc__.split('\n\n')[3]
+        self.assertEqual(exp_split_ints_reuturn, split_ints_return)
+
+        optional_params = (
+            identity_with_optional_metadata.__call__.__doc__.split('\n\n')[2])
+        self.assertEqual(exp_optional_params, optional_params)
+
+        no_input_method = no_input_method.__call__.__doc__
+        self.assertEqual(exp_no_input_method, no_input_method)
+
+        params_only = params_only_method.__call__.__doc__
+        self.assertEqual(exp_params_only, params_only)
+
+
+exp_merge_calldoc = """\
+Merge mappings
+
+This method merges two mappings into a single new mapping. If a key is \
+shared between mappings and the values differ, an error will be raised.
+
+Parameters
+----------
+mapping1: Mapping
+    Mapping object to be merged
+mapping2: Mapping
+    No description available
```

Comment:
Descriptions aren't required in numpydoc, so it's probably better to just omit them if not set by the plugin.